src/ring_theory/dedekind_domain/ideal.lean

```diff
@@ -902,8 +902,9 @@ open_locale classical
 lemma ideal.count_normalized_factors_eq {p x : ideal R} (hp0 : p ≠ ⊥) [hp : p.is_prime] {n : ℕ}
   (hle : x ≤ p^n) (hlt : ¬ (x ≤ p^(n+1))) :
   (normalized_factors x).count p = n :=
-count_normalized_factors_eq ((ideal.prime_iff_is_prime hp0).mpr hp).irreducible (normalize_eq _)
-  (ideal.dvd_iff_le.mpr hle) (mt ideal.le_of_dvd hlt)
+count_normalized_factors_eq ((ideal.prime_iff_is_prime hp0).mpr hp).irreducible
```

Comment:
Maybe add a comment explaining that this otherwise style-breaking code is done to avoid a timeout?

Reply:
Good idea; note (warning) added.